R/profile_wrapper.R

```diff
@@ -79,16 +88,18 @@ profile_wrapper <- function(mydir, model_settings) {
 
     if (sum(check_para) == 0) {
       print(para)
-      stop(paste0("The input profile_custom does not match a parameter in the control.ss_new file."))
+      stop("The input profile_custom does not match a parameter in the file ",  
```

Comment:
I would remove "the file" because it is clear that the object in `model_settings$oldctlfile` is a file.